### PR TITLE
[STORY-21] Create Login/Registration and Home Pages and Routing Between Pages

### PR DESCRIPTION
**Pull request by sdcarlson**
This PR sets up the following routes:
- '/': login/registration page
- '/home': home page for selecting between 3 actions

Note: "Edit Saved Servers" option on home page opens up a dropdown menu

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Discord Webapp to Configure Servers",
   "main": "index.js",
   "scripts": {
-    "start": "webpack | babel-node src/server.js",
```
Comment (gyu123987) [inline webapp/package.json:7]: add webpack back

File: webapp/package.json [hunk 2]
```diff
@@ -17,7 +17,11 @@
     "express": "^4.18.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+<<<<<<< HEAD
     "uuidv4": "^6.2.13"
+=======
```
Comment (gyu123987) [inline webapp/package.json:22]: fix conflict

File: webapp/package.json (hunk 1 above)
Comment (sdcarlson) [reply webapp/package.json]: done

File: webapp/package.json (hunk 2 above)
Comment (sdcarlson) [reply webapp/package.json]: done

**Review by gyu123987 — APPROVED** [review]

**Pull request merged**